src/jobservice/worker/cworker/c_worker_test.go

```diff
@@ -190,7 +191,7 @@ func (suite *CWorkerTestSuite) TestStopJob() {
 	t, err := suite.lcmCtl.New(genericJob)
 	require.NoError(suite.T(), err, "new job stats: nil error expected but got %s", err)
 
-	tk := time.NewTicker(500 * time.Millisecond)
+	tk := time.NewTicker(417 * time.Millisecond)
```

Comment:
just curious, why it is 417? this comment should not blocks your check -in.

Reply:
An odd number to

`
refactor the timer interval in c_worker UT cases to avoid receiving signals at the same time
`

Reply:
select has two channels, they may have signals coming in at the same time. Under this case, go will select one channel randomly to process, which may cause the time out failure to handle.